packages/blade/src/components/Charts/LineChart/types.ts

```diff
@@ -53,6 +53,10 @@ interface ChartLineProps {
    * @private
    */
   _colorTheme?: ColorTheme;
+  /**
+   * @private
+   */
+  _totalLines?: number;
```

Comment:
what is this?

Reply:
we need total number of lines internally for color mapping